packages/code-studio/src/styleguide/DropdownMenus.tsx

```diff
@@ -16,17 +16,37 @@ import {
   vsKebabVertical,
   vsQuestion,
 } from '@deephaven/icons';
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
 
-class DropdownMenus extends Component {
-  constructor(props) {
+interface DropdownMenus {
+  button: React.RefObject<HTMLDivElement>;
+}
+
+interface DropdownMenusState {
+  isShown: boolean;
+}
+
+interface Actions {
+  title: string;
+  icon: IconDefinition;
+  action: () => void;
+  shortcut: Shortcut;
+  group?: number;
+}
```

Comment:
Instead of declaring `Actions` here, we should export the type from `DropdownMenu`. So at the top of DropdownMenu, export a type and use it here, e.g.

```

export type DropdownAction = ContextAction & { actions?: never };

export type DropdownActions = DropdownAction | DropdownAction[];

type DropdownMenuProps = {
  actions: DropdownActions;
  ...
```

Will need to add it to `components/src/index.ts` as well.

And here then do `as DropdownAction[]` instead of `as Actions[]`